lib/providers/password.js

```diff
@@ -16,31 +16,46 @@ var errors = require('../errors');
 function PasswordProvider (client, options) {
     // Jar for storing the session cookie
     this.jar = request.jar();
+    // storage of the CSRF Token as given in the response from the server
+    this.csrfToken = null;
+    this.csrfTokenLocation = 'x-csrf-token';
```

Comment:
To be clear the update document is a spec of our implementation of CSRF cookies. If we introduce a change to the header, it would probably be a behavioural change and not just a renaming.